src/gen/haClient/base.ts

```diff
@@ -17,7 +17,7 @@ import { Configuration } from './configuration'
 // @ts-ignore
 import globalAxios, { AxiosPromise, AxiosInstance, AxiosRequestConfig } from 'axios'
 
-export const BASE_PATH = process?.env?.REACT_APP_API_URL!.replace(/\/+$/, '')
+export const BASE_PATH = 'https://api-dev.hei.school'.replace(/\/+$/, '')
```

Comment:
C2. bring process.env.REACT_APP_API_URL!.replace(/\/+$/, '')
to guarantee that the deployed version will not crash as this API_URL is dynamic

instead, adjust your .env to 'https://api-dev.hei.school' locally